tests/backup/backup_test.go

```diff
@@ -2008,3 +2112,68 @@ func validateBackupCluster() bool {
 	}
 	return true
 }
+
+func createRuleForBackup(rule_name string, orgID string, pod_selector []map[string]string, action_value []string,
```

Comment:
create common file and move this function to that

Reply:
done